src/components/Project/ProjectTabs/CyclesPayoutsTab/UpcomingCycleChangesCallout.tsx

```diff
@@ -1,27 +1,14 @@
 import { InformationCircleIcon } from '@heroicons/react/24/outline'
 import { t } from '@lingui/macro'
-import { useMemo } from 'react'
 import { twMerge } from 'tailwind-merge'
-import { useCurrentUpcomingConfigurationPanel } from '../hooks/useConfigurationPanel/useCurrentUpcomingConfigurationPanel'
 
-export const UpcomingCycleChangesCallout = () => {
-  const data = useCurrentUpcomingConfigurationPanel('upcoming')
-
-  const loading = useMemo(
-    () =>
-      Object.values(data).some(value =>
-        Object.values(value ?? {}).some(v => v.new === undefined),
-      ),
-    [data],
-  )
-
-  const hasChanges = useMemo(
-    () =>
-      Object.values(data).some(value => {
-        return Object.values(value ?? {}).some(v => v.old !== v.new)
-      }),
-    [data],
-  )
+export const UpcomingCycleChangesCallout = ({
+  hasChanges,
+  loading,
+}: {
+  hasChanges: Boolean,
+  loading: Boolean
+}) => {
   const text = hasChanges
     ? t`This cycle has upcoming changes`
```

Comment:
This copy will need to change in V4. Maybe we just don't make this component shared? Or make the banner Callout shared but not the copy?

Reply:
Got it, will fix this.